engine-tests/src/tests/uniswap.rs

```diff
@@ -55,13 +55,21 @@ fn test_uniswap_exact_output() {
         context.add_equal_liquidity(LIQUIDITY_AMOUNT.into(), &token_a, &token_b);
     test_utils::assert_gas_bound(profile.all_gas(), 126);
     let wasm_fraction = 100 * profile.wasm_gas() / profile.all_gas();
-    assert!(wasm_fraction >= 23, "{}% not more than 23%", wasm_fraction);
+    assert!(
+        15 <= wasm_fraction && wasm_fraction <= 20,
```

Comment:
In terms of absolute cost, this went down from `125965273627830` to `116478702274944` :tada: